src/khoj/main.py

```diff
@@ -137,6 +141,12 @@ def start_server(app, host=None, port=None, socket=None):
     if socket:
         uvicorn.run(app, proxy_headers=True, uds=socket, log_level="debug", use_colors=True, log_config=None)
     else:
+        url = f"http://{host}:{port}"
+        logger.info(f"🌗 Khoj is running at {url}")
+        try:
+            webbrowser.open(url)
+        except:
+            logger.warning("🚧 Unable to open browser. Please open it manually to configure Khoj.")
```

Comment:
Should this only be shown in GUI mode? Obsidian and Emacs users do not need the server to open the browser. They are already on the interfaces they'll be interacting with Khoj from